### PR TITLE
feat: NavSatFix publisher for GNSS simulation

### PR DESCRIPTION
**Pull request by Autumn60**
- add geographic coordinate calculation scripts (`GeoCoordinate.cs`, `GeoCoordinateConverter.cs`)
- add parameter that can be set from the Inspector window for geographic calculation to `Environment.cs`
- add `GnssRos2NavSatFixPublisher.cs` to publish `sensor_msgs/NavSatFix` topic
To minimize impact on other projects, it's implemented separately from the existing GnssRos2Publisher.

The motivation is to test `eagleye` at runtime with AWSIM.

ref link:
Geographpic coordinate calculations are implemented with reference to the following sites
=> https://vldb.gsi.go.jp/sokuchi/surveycalc/main.html

### CODE REVIEW

File: Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinate.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+namespace AWSIM.Geographic
+{
+[System.Serializable]
+public class GeoCoordinate
+{
+  public double Latitude;
+  public double Longitude;
+  public double Altitude;
```
Comment (mackierx111) [inline Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinate.cs:8]: Is it immutable? If it's immutable, I think it should be a getter property.

File: Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinateConverter.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
+using System;
+using UnityEngine;
+
+namespace AWSIM.Geographic
+{
+public static class GeoCoordinateConverter
```
Comment (mackierx111) [inline Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinateConverter.cs:6]: Please add a brief comment on the calculation method used in this class. For example, post the URL of the site you referred to.

**Review by mackierx111 — CHANGES_REQUESTED** [review]
good implementation! Please check out the comments.

File: Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinate.cs (hunk 1 above)
Comment (Autumn60) [reply Assets/AWSIM/Scripts/GeoCoordinate/GeoCoordinate.cs]: That is certainly true.
I'll fix it to use a getter property.

**Comment by Autumn60** [comment]
@mackierx111 
Thank you for your accurate point.
I've corrected the relevant section.

**Review by mackierx111 — APPROVED** [review]
LGTM👍

**Pull request merged**